staubli_rx160_support/urdf/rx160_macro.xacro

```diff
@@ -2,6 +2,14 @@
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
   <xacro:include filename="$(find staubli_resources)/urdf/common_materials.xacro"/>
 
+  <property name="base_mass"   value="50.0" />
+  <property name="link_1_mass" value="30.0" />
+  <property name="link_2_mass" value="20.0" />
+  <property name="link_3_mass" value="20.0" />
+  <property name="link_4_mass" value="10.0" />
+  <property name="link_5_mass" value="1.0" />
+  <property name="link_6_mass" value="0.05" />
```

Comment:
I was experimenting with mass values, so it was easier to keep them in one place. I got the values extracted from the CAD models using Solidworks now, so they should not change anymore, and therefore I removed those properties in my latest commit.